templates/website/index-index.html

```diff
@@ -3,15 +3,14 @@
 template_draw('header', $values);
 ?>
 
-<div class="row row-full-width hero">
+<div class="row row-full-width hero hero--responsiveness">
     <div class="large-10 large-centered columns">
         <a href="/" class="wtt-logo"><img src="/static/img/logo.png" width="250" height="70" alt="WriteToThem"></a>
         <a href="/about-qa" target="_blank" class="wtt-help" title="Opens in a new window">Help</a>
         <div style="text-align:center;">
-            <?=$values['blurb-top'] ?>
-            <?php if ($values['num_messages']) {
-                echo "<!-- * $values[cobrand] messages: $values[num_messages] -->";
-            } ?>
```

Comment:
There's no reason to remove this if statement; it should be left in for any subdomain users who are using it to see the number of messages.